src/main/resources/css/colorPickMacro.css

```diff
@@ -0,0 +1,44 @@
+.background-image {
+    z-index: 0;
+    opacity: 0.4;
+    min-width: 600px;
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    background-size: 100% 100%;
+    background-repeat: no-repeat;
+}
+
+#main {
+    z-index: 1;
+}
+
+.field-group {
```

Comment:
`.field-group` class is one of the default css classes in Atlassian AUI. I think it's not a good idea to override its styles.
why not to define your own class?
the same is true for `#main`, `.btn` and `.background-image`

Reply:
`.background-image` isw my own class. There is one `<div>` with this class, generated via JQuery

Reply:
also i have to edit `#main`, because if I don't set `z-index` to `1`, background image will be overflow page view